modules/plugins/grid.js

```diff
@@ -0,0 +1,95 @@
+/* @flow */
+
+function isSimplePositionValue(value: any) {
+  return typeof value === 'number' && !isNaN(value)
```

Comment:
Can we use `Number.isNaN` instead of this function? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN

Reply:
why? `isNaN` is available in more places, and the `typeof` ensures it'll behave identically.

Reply:
Yeah we came full circle on this one